flink-table/flink-table-planner/src/main/scala/org/apache/flink/table/planner/plan/rules/logical/LogicalWindowAggregateRuleBase.scala

```diff
@@ -98,7 +99,7 @@ abstract class LogicalWindowAggregateRuleBase(description: String)
     // we don't use the builder here because it uses RelMetadataQuery which affects the plan
     val newAgg = LogicalAggregate.create(
       newProject,
-      agg.indicator,
+      ImmutableList.of[RelHint](),
```

Comment:
Current call of `LogicalAggregate#create` https://github.com/apache/calcite/blob/cd5229b9a371e20de207231d55bcbddf1ff39ec2/core/src/main/java/org/apache/calcite/rel/logical/LogicalAggregate.java#L87-L91 is identical to this one with empty hint list https://github.com/apache/calcite/blob/cd5229b9a371e20de207231d55bcbddf1ff39ec2/core/src/main/java/org/apache/calcite/rel/logical/LogicalAggregate.java#L64-L73